pages/docs/quickstart.mdx

```diff
@@ -54,13 +54,20 @@ To do this, you need a GitHub account. If you do not have one yet, you can creat
 
 <ClipboardCopy>
 ```js
+{
+  "name": "Sara Jaoude",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Lawyer turned **Digital Nomad**. Travel enthusiast Yoga & Peloton aficionado",
+  "avatar": "https://github.com/SaraJaoude.png",
   "links": [
     {
       "name": "Follow me on Twitter",
       "url": "https://twitter.com/SaraJaoude",
       "icon": "FaTwitter"
-    }
+    },
```

Comment:
I think this trailing comma makes the json invalid?

Reply:
Yep, it makes it invalid, unless there's another field next to it.

Reply:
ok I will remove this